packages/core/src/caches/index.ts

```diff
@@ -56,6 +45,112 @@ export class RedisCache implements CacheStore {
       consoleLog.info('[CACHE] Disconnected from Redis');
     }
   }
+
+  protected getSocketOptions(url: URL) {
```

Comment:
will it be better to have a return type here?

Reply:
`RedisSocketOptions` type is not exported from `redis` package, how should I proceed?